### DISCUSSION TITLE
SubmitFeed de Livros POST_FLAT_FILE_BOOKLOADER_DATA

### DISCUSSION THREAD

**Discussion by AGL15** [abuzuhri/Amazon-SP-API-CSharp#335] [General]
Hi, first of all thank you very much and congratulations for the work.
How do I upload a TXT of Books (Feeds SP-API  https://developer-docs.amazon.com/sp-api/docs/feeds-api-v2021-06-30-reference)?

Below is an attached file that I currently send through the MWS API that will be discontinued at the end of the year.

[Flat.File.BookLoader.br.txt](https://github.com/abuzuhri/Amazon-SP-API-CSharp/files/9649734/Flat.File.BookLoader.br.txt)

**Comment by abuzuhri** [comment]
@AGL15 do you see sample feed upload ?

**Comment by AGL15** [comment]
Hi, I don't know how to send. I would like to know (code example) how to send (upload) the TXT file attached above.

PS: Translated by Google Translate.

**Comment by AGL15** [comment]
If anyone can include in the project upload (register and update) Book, Magazine, Comic Book, CD, DVD and Vinyl Record (new and used) I would appreciate it.

PS: I can pay to contribute to the project.
